lerna.json

```diff
@@ -1,8 +1,8 @@
 {
   "lerna": "2.9.0",
   "version": "1.11.3",
-  "npmClient": "yarn",
   "packages": [
+    "example",
```

Comment:
Why do we want the example/ folder treated as a package?

Reply:
See this comment: https://github.com/nasa/cumulus/pull/871#discussion_r265273132